conftest.py

```diff
@@ -71,3 +75,41 @@ def pytest_ignore_collect(path, config):
             # If the current Python version does not meet the minimum required, skip this directory
             if sys.version_info[0:2] < min_required:
                 outcome.force_result(True)
+
+
+@pytest.fixture(scope="module")
```

Comment:
I'm not sure if this can be overridden if it's desired to use the fixture on a per-function/class scope. I imagine that would be a valid use-case at some point but isn't something we need now.